week4/src/pages/pokesearch.tsx

```diff
@@ -0,0 +1,96 @@
+import React, {ChangeEvent, FormEvent, useState, useEffect} from 'react'
+import Search from './components/search'
+import DisplayTable from './components/displaytable'
+
+function PokeSearch()
+{
+
+    const [search, setSearch] = useState("")
+
+    const [loading, setLoading] = useState(true)
+
+    const [pokemonList, setPokemonList] = useState<Record<string, any>[]>([])
```

Comment:
you can replace any with a Pokemon interface, create an interface in the schema folder as shown below
```
export interface Pokemon {
  name: string,
  id: number,
  ....
}
```
Then replace any with `Pokemon`
`const [pokemonList, setPokemonList] = useState<Pokemon[]>([])`